src/PiggyCustomEnchants/Main.php

```diff
@@ -460,9 +460,10 @@ public function addEnchantment(Item $item, $enchants, $levels, $check = true, Co
                 $levels[] = 1;
             }
         }
-        $combined = array_combine($enchants, $levels);
+        $i = 0;
```

Comment:
Why???

Reply:
`Object of class PiggyCustomEnchants\CustomEnchants\CustomEnchants could not be converted to string`

Reply:
That should not be causing the issue...